crates/broker/src/config.rs

```diff
@@ -126,6 +126,14 @@ pub struct MarketConf {
     ///
     /// Maximum number of concurrent lockin requests that can be processed at once
     pub max_concurrent_locks: Option<u32>,
+    /// Optional cache directory for storing downloaded images and inputs
+    /// if not set, files will be re-downloaded every time
+    pub cache_dir: Option<PathBuf>,
```

Comment:
This used to be part of `Args`, but after the refactoring access to the args is more difficult, also conceptually it is only used in the marked and thus should probably belong to the market config.